configure.ac

```diff
@@ -245,6 +245,10 @@ if test x$enable_harfbuzz = xyes; then
 			# for uniscribe support:
 			TTF_LIBS="$TTF_LIBS -lusp10 -lgdi32 -lrpcrt4"
 			PC_LIBS="$PC_LIBS -lusp10 -lgdi32 -lrpcrt4"
+          ;;
+		  *)
+		    # compiler might optimize sinf+cosf into sincosf
+		    TTF_LIBS="$TTF_LIBS -lm"
```

Comment:
Maybe conditionalize this to at least `enable_harfbuzz_builtin` ?

Reply:
I think this is already in the "`x$enable_harfbuzz_builtin = xyes` context".

Reply:
My bad. Sorry for the noise.

Reply:
Nah, I'm better at making noise :P